### PR TITLE
ssl update

### PR DESCRIPTION
**Pull request by chongjunkim**
## Is this a public release?
* meta.user.js version numbers updated to next release number? yes
* meta.js version numbers updated to next release number? yes

## Is this a pre-release?
* develop.meta.user.js version numbers updated to next release number? no

### CODE REVIEW

File: assets/js/meta.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-// @version 1.1.8
+// @version 1.1.9
 // @run-at document-end
 // @downloadURL https://github.com/cirept/autofillReplacer/raw/master/assets/js/meta.user.js
 // @resource toolStyles https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/assets/css/autofill.css
 // @resource changeLog https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/docs/LatestChanges.md
 // @resource toolInstructions https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/docs/README.md
-// @include http://websites.cobalt.com/wsm/editSite.do*
-// @include http://websites.cobalt.com/wsm/cms.do*
+// @include https://websites.cobalt.com/wsm/editSite.do*
```
Comment (cirept) [inline assets/js/meta.js:11]: include the http version of the url to cover any issues with someone using the http version of the site.

File: assets/js/meta.user.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-// @version 1.1.8
+// @version 1.1.9
 // @run-at document-end
 // @downloadURL https://github.com/cirept/autofillReplacer/raw/master/assets/js/meta.user.js
 // @resource toolStyles https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/assets/css/autofill.css
 // @resource changeLog https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/docs/LatestChanges.md
 // @resource toolInstructions https://cdn.rawgit.com/cirept/autofillReplacer/1.1.8/docs/README.md
-// @include http://websites.cobalt.com/wsm/editSite.do*
-// @include http://websites.cobalt.com/wsm/cms.do*
+// @include https://websites.cobalt.com/wsm/editSite.do*
```
Comment (cirept) [inline assets/js/meta.user.js:11]: include the http version of the url to cover any issues with someone using the http version of the site.

**Review by cirept — CHANGES_REQUESTED** [review]

File: assets/js/meta.js (hunk 1 above)
Comment (chongjunkim) [reply assets/js/meta.js:11]: I added the http version and pushed a commit.

File: assets/js/meta.user.js (hunk 2 above)
Comment (chongjunkim) [reply assets/js/meta.user.js:11]: I added the http version and pushed a commit.

**Review by chongjunkim — COMMENTED** [review]
I added the http version and pushed a commit.

**Review by cirept — APPROVED** [review]

**Pull request merged**